### PR TITLE
changes to support cacheExpirationTimeInMinutes=0, if user wants to disable cache

### PR DESCRIPTION
**Pull request by suryasoma**
Fix to the issue: https://github.com/GoogleCloudDataproc/spark-bigquery-connector/issues/593

changes to support cacheExpirationTimeInMinutes=0, if user wants to disable cache
Added unit test
Ran integration tests

### CODE REVIEW

File: spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/SparkBigQueryConfigTest.java [hunk 1]
```diff
@@ -190,6 +190,29 @@ public void testConfigFromOptions() {
     assertThat(config.getBigQueryJobLabels()).containsEntry("foo", "bar");
   }
 
+  @Test
```
Comment (VinayLondhe14) [inline spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/SparkBigQueryConfigTest.java:193]: Can you add another test to make sure an exception is thrown if a negative value is passed? Optionally, you can also add a test to make sure that a positive value passes

**Review by VinayLondhe14 — APPROVED** [review]
